src/main/java/ru/sbtqa/tag/swingback/Bridge.java

```diff
@@ -0,0 +1,18 @@
+package ru.sbtqa.tag.swingback;
+
+import java.util.Locale;
+
+/**
+ * Class for transferring data to remote jvm
+ */
+public class Bridge {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)